website/docs/docs/cloud/dbt-cloud-ide/ide-glossary.md

```diff
@@ -0,0 +1,155 @@
+---
+title: "IDE glossary"
+id: ide-glossary
+description: "Develop, test, run, and build in the Cloud IDE. With the Cloud IDE, you can compile dbt code into SQL and run it against your database directly"
+sidebar_label: IDE glossary
+tags: [IDE]
+---
+
+The [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud) is an interface tool for developers to effortlessly build, test, run, and version-control their dbt projects, and enhance data governance — all from the convenience of your browser. Use the Cloud IDE to compile dbt code into SQL and run it against your database directly -- no command line required!
+
+This page offers comprehensive definitions and terminology of user interface elements, allowing you to navigate the IDE landscape with ease.
+
+## Basic layout
+
+The IDE is designed to streamline your workflow, and features a popular user interface and layout with all files and folders on the left, editor on the right, and command and console information at the bottom. 
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/ide-basic-layout.jpg" width="100%" title="The Cloud IDE layout includes version control on the upper left, files/folders on the left, editor on the right and command/console at the bottom"/>
+
+The basic layout of the IDE landing page with the following areas:
+
+1. **Git repository link &mdash;** Clicking the Git repository link, located on the upper left of the IDE, takes you to your repository on the same active branch. 
+    * **Note:** This feature is only available for GitHub or GitLab repositories on multi-tenant dbt Cloud accounts.
+
+2. **Documentation site button &mdash;** Clicking the Documentation site book icon, located next to the Git repository link, leads to the dbt Documentation site. The site is powered by the latest dbt artifacts generated in the IDE using the `dbt docs generate` command from the Command bar.
+
+3. [**Version Control**](#editing-features) &mdash; The IDE's powerful Version Control section contains all git-related elements, including the Git actions button and the **Changes** section. 
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/ide-upper-left.jpg" width="40%" title="The Git repo link, documentation site button, and Version Control menu"/>
```

Comment:
Being that the numbers capture this diagram, I suggest moving it up before the numbered paragraphs. You could put a zoom in of the file explorer before step 4.